core/services/promreporter/prom_reporter_test.go

```diff
@@ -38,7 +38,7 @@ func newLegacyChainContainer(t *testing.T, db *sqlx.DB) legacyevm.LegacyChainCon
 	ethClient := evmtest.NewEthClientMockWithDefaultChain(t)
 	estimator := gas.NewEstimator(logger.TestLogger(t), ethClient, config, evmConfig.GasEstimator())
 	lggr := logger.TestLogger(t)
-	lp := logpoller.NewLogPoller(logpoller.NewORM(testutils.FixtureChainID, db, lggr, pgtest.NewQConfig(true)), ethClient, lggr, 100*time.Millisecond, false, 2, 3, 2, 1000)
+	lp := logpoller.NewLogPoller(logpoller.NewORM(testutils.FixtureChainID, db, lggr), ethClient, lggr, 100*time.Millisecond, false, 2, 3, 2, 1000)
```

Comment:
Without `pgtest.NewQConfig(true)`, how are we communicating that we want to log SQL?

Reply:
I don't see how logging SQL in some test cases benefits the tests. I made the decision to remove this to clean things up. If a test fails then the require/ asserts should be sufficient to figure out why. I think if looking at the SQL logging is required, then the direct ORM tests should not pass. You can also see any SQL errors anyways when making a call to the ORM so logging seems unnecessary.

Reply:
I wasn't thinking about tests. Haven't you also removed this feature from the production code?

Reply:
Seems to be set false everywhere except for some testing as far as I can see

Reply:
This is a major feature and it is user configurable in production. We cannot drop it without an equivalent replacement.

Reply:
I'll create a PR branching off this to add logging back in.